client/homebrew/brewRenderer/toolBar/toolBar.jsx

```diff
@@ -214,7 +247,18 @@ const ToolBar = ({ displayOptions, currentPage, totalPages, onDisplayOptionsChan
 					className='tool'
 					type='button'
 					title='Next Page(s)'
-					onClick={()=>scrollToPage(pageNum + 1)}
+					onClick={()=>{
+						// if there are multiple pages in a 'row' and they are in 'view',
+						// then the 'max'/last page in view will always be the same, and
+						// the other pages will always be the same (since the viewport doesn't change).
+						// So this needs to scroll to the 'max', then see what is newly in view,
+						// and if the same pages are visible, do it again but +1.
+						const start = _.max(visiblePages);
+						scrollToPage(start);
+						if(start === _.max(visiblePages)){
+							scrollToPage(start + 1);
+						};
+					}}
 					disabled={pageNum >= totalPages}
```

Comment:
as previously mentioned, this needs updating.

Reply:
...done